docs/source/user_guide/getting_started_with_dpf_server.rst

```diff
@@ -64,14 +64,14 @@ With PyDPF-Core, you can also explicitly start a DPF Server using:
     server = dpf.start_local_server()
 
 To start a DPF Server from outside a Python environment, you can also use the execution script provided with your DPF Server package.
-On Windows, start the DPF Server by running the Ans.Dpf.Grpc.bat file in the unzipped package.
-On Linux, start the DPF Server by running the Ans.Dpf.Grpc.sh file in the unzipped package.
+On Windows, start the DPF Server by running the ``Ans.Dpf.Grpc.bat`` file in the unzipped package.
+On Linux, start the DPF Server by running the ``Ans.Dpf.Grpc.sh`` file in the unzipped package.
 
 Running the DPF Server in a Docker container
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Along with the ansys_dpf_server_lin_v2023.2.pre0.zip archive mentioned in :ref:`Installing DPF Server <target_installing_server>`, download the Dockerfile.
-2. Copy both the archive and Dockerfile in a folder and navigate into that folder.
+1. Along with the ansys_dpf_server_lin_v2023.2.pre0.zip archive mentioned in :ref:`Installing DPF Server <target_installing_server>`, download the ``Dockerfile``.
```

Comment:
@PProfizi Is the name of the file ``Dockerfile``? If so, you have written and tagged it correctly. If it is a generic reference, it should be "download the Docker file" with it being two words--a file belonging to or related to Docker.

Reply:
@PipKat yes indeed the file is named Dockerfile